### PR TITLE
Trying to solve the newline problem

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

**Comment by bdunne** [comment]
cc @carbonin & @Fryguy

File: lib/extensions/ar_migration.rb [hunk 1]
```diff
@@ -52,11 +28,41 @@ def self.update_local_migrations_ran(version, direction)
         WHERE id = #{region.id}
       SQL
     end
+
+    class RemoteRegionThing
```
Comment (Fryguy) [inline lib/extensions/ar_migration.rb:32]: You are the master of names LOL

File: lib/extensions/ar_migration.rb [hunk 2]
```diff
@@ -52,11 +28,41 @@ def self.update_local_migrations_ran(version, direction)
...
+      attr_reader :region, :subscription, :version
+      def initialize(subscription, version)
+        @region       = MiqRegion.find_by(:region => subscription.provider_region)
```
Comment (Fryguy) [inline lib/extensions/ar_migration.rb:35]: Avoid using model in migration (same reasons for why we don't allow them in manageiq-schema)

Comment (carbonin) [reply lib/extensions/ar_migration.rb:35]: I'm not sure what the alternative would be here. Should we do something like the stubs we use for migrations? We're not using any model methods, all we need it for is to query the `migrations_ran` attribute.

File: lib/extensions/ar_migration.rb (hunk 1 above)
Comment (bdunne) [reply lib/extensions/ar_migration.rb:32]: :laughing: I changed it like 4 times before giving up and using this just so I could continue re-working the code.

File: lib/extensions/ar_migration.rb (hunk 2 above)
Comment (bdunne) [reply lib/extensions/ar_migration.rb:35]: Something like this maybe?
```ruby
module ArPglogicalMigration
  class NotMiqRegion
    self.table_name = "miq_regions"
ennd
```

Comment (Fryguy) [reply lib/extensions/ar_migration.rb:35]: yeah?  or just query it raw?

**Pull request closed without merging**